src/components/EmailSubscription/index.view.tsx

```diff
@@ -1,9 +1,75 @@
-import React from "react";
+import React, { useState } from "react";
+import axios, { AxiosResponse } from "axios";
+import { ReactComponent as Arrow } from "images/arrow.svg";
+
 import "./EmailSubscription.scss";
 
 const EmailSubscriptionForm: React.FC = () => {
+  const [userEmail, setUserEmail] = useState("");
+  const [feedbackMessage, setFeedbackMessage] = useState("");
+  const [showFeedback, setShowFeedback] = useState(true);
+  const [showInput, setShowInput] = useState(true);
+
+  const handleInput = (event: React.FormEvent<HTMLInputElement>) => {
+    setUserEmail(event.currentTarget.value.trim());
+  };
+
+  const handleKey = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === "Enter") {
+      submitEmail();
+    }
+  };
+
+  const submitEmail = () => {
+    setShowFeedback(true);
+    setFeedbackMessage("Please wait while your message is being submitted...");
+    setUserEmail("");
+
+    axios
+      .post(process.env.REACT_APP_SUBSCRIBE_ENDPOINT || "", {
```

Comment:
I'd create a variable for this and instead of just have the variable value in here. 

```tsx
const subscribeEndpoint = process.env.REACT_APP_SUBSCRIBE_ENDPOINT || ""
...
axios.post(subscribeEndpoint)...
```

This will allow for cleaner and more readable code.